js/components/moter/DialogmoterInnholdLenke.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+const DialogmoterInnholdLenke = (
+    {
+        imgUrl,
+        visning,
+        innholdstekster,
+        minimal = true,
+        rootUrl,
+    }) => {
+    if (minimal) {
```

Comment:
Ville laget dette som to helt separate komponenter og minimal-lagt sjekken utenfor.